src/pages/trade/VerticalPanel.tsx

```diff
@@ -23,6 +24,32 @@ const HISTOGRAM_SIDES_BY_LAYOUT = {
 export const VerticalPanel = ({ tradeLayout }: { tradeLayout: TradeLayouts }) => {
   const stringGetter = useStringGetter();
   const [value, setValue] = useState(Tab.Orderbook);
+  const [rowsPerSide, setRowsPerSide] = useState<number | undefined>(undefined);
```

Comment:
This should use one of our libraries' resize observer components or hooks. This kind of code is too finnicky to do manually imo.

Reply:
maybe https://github.com/ZeeCoder/use-resize-observer/. But then again, if this works consistently maybe library isn't necessary

Reply:
Resize observers have the advantage of directly watching the size of the element so we know it will trigger if any layout or page change causes the reflow.

Reply:
yeahhh so I did take a look at that; hmm I think integrating ResizeObserver would also unlock some other features (like being able to resize the bottom positions table). Was hoping for a quick alternative we could get out in the meantime because I think using the resize observer might require restructuring the format of the page but maybe this is too jank to merge 😢

Reply:
oh actually you're right, resizeobserver is way better for this - pushing an update!